frontend/src/lib/components/button/secondaryButton/SecondaryButton.svelte

```diff
@@ -0,0 +1,80 @@
+<script lang="ts">
+  import {Icon} from '$lib/components/icon';
+  import {ButtonBase} from '../base';
+  import type {SecondaryButtonProps} from './SecondaryButton.type';
+
+  type $$Props = SecondaryButtonProps;
+
+  export let icon: $$Props['icon'];
+  export let text: $$Props['text'];
+  export let iconPos: $$Props['iconPos'] = 'left';
+  export let color: $$Props['color'] =
```

Comment:
why does the color depend on the position?

Reply:
Good point! It was related to the different default styling between the button types, but it's not very logical, so I'll change color to always default to `'primary'`.